spec/unit/multi_select_spec.rb

```diff
@@ -375,4 +375,42 @@ def output_helper(prompt, choices, active, selected, options = {})
       "What letter? \e[32mA\e[0m\n\e[?25h"
     )
   end
+
+  context "with filter" do
```

Comment:
I would encourage you to make specs more consistent with already present either by using the `output_helper` or manually creating the output string. Currently these fail on Windows.